### PR TITLE
Add workflows for screenbuilder commands 

### PR DESCRIPTION
**Pull request by Mitko-Kerezov**
Using the **command-help.json** file located inside the resources folder a tip can be shown after specific commands to help guide the user into successfully creating a project.
Added command-help.json with sample data inside.
Implements [#293553](http://teampulse.telerik.com/view#item/293553)
Merge after https://github.com/telerik/mobile-cli-lib/pull/427

### CODE REVIEW

File: lib/cordova-resource-loader.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+///<reference path=".d.ts"/>
+"use strict";
+
+import path = require("path");
```
Comment (teobugslayer) [inline lib/cordova-resource-loader.ts]: Consider using ES6 import

**Comment by teobugslayer** [comment]
run ci

**Comment by Mitko-Kerezov** [comment]
Ping @teobugslayer @Fatme I think this is the final revision :smiley_cat:

**Comment by Mitko-Kerezov** [comment]
run ci

**Comment by teobugslayer** [comment]
:+1:

**Pull request merged**